x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph/graph_popover.stories.tsx

```diff
@@ -179,9 +179,11 @@ const Template: Story = () => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const nodeClickHandler = (...args: any[]) => popoverOpenWrapper(nodePopover.onNodeClick, ...args);
 
-  const nodes: EntityNodeViewModel[] = useMemo(
-    () =>
-      (['hexagon', 'ellipse', 'rectangle', 'pentagon', 'diamond'] as const).map((shape, idx) => ({
+  const nodes: Array<EntityNodeViewModel | LabelNodeViewModel> = useMemo(
```

Comment:
Well in the case of stories, I don't see any harmful results in doing so.

Besides that, I kept it to the cases when I wished to optimize the amount of renderings in our graph. So I thought it validated that the stories were not causing any re-rendering.

Reply:
🤦 I did not realize this was a Storybook story, however,  this is a good example of how storybook stories are easier to review for testing purposes.

One of the things I wanted to do for On Week,  was introduce using Storybook stories for Jest testing,  it may make checking for performance like re-rendering easier.

Reply:
I have an example you can take a look into: https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_layout.stories.test.tsx